.vale/fixtures/RedHat/ObviousTerms/testinvalid.adoc

```diff
@@ -0,0 +1,3 @@
+//vale-fixture
```

Comment:
The Red Hat rule set works by linting line by line. This `//vale-fixture` comment is only required in the rule sets where we don't have one error per line and we need some way to count the errors in the test fixtures.

Reply:
Ah, OK. So I should remove this comment.